pkg/cmd/pulumi/state_move.go

```diff
@@ -35,16 +35,18 @@ import (
 )
 
 type stateMoveCmd struct {
-	Stdin     io.Reader
-	Stdout    io.Writer
-	Colorizer colors.Colorization
-	Yes       bool
+	Stdin          io.Reader
+	Stdout         io.Writer
+	Colorizer      colors.Colorization
+	Yes            bool
+	IncludeParents bool
```

Comment:
Only minor thing I'm wondering is if we should call it `TargetParents` to match `TargetDependents`/the general use of "target" as the verb. But minor and happy either way. Also I feel like we may have already had that discussion and I quite possibly supported "include", so sorry if so :sweat_smile:

Reply:
I'm not sure.  I think `--include-parents` makes sense especially if we want to introduce a flag like `--exclude-children`.  Not sure what we would name that if we could call this flag `--target-parents`. But I can see the argument for calling it `TargetParents`. I don't have a strong opinion here tbh.

Reply:
I think the point about exclude is a great one. Happy to leave as-is :+1: Thanks for tolerating the bikeshedding!